doc/contributor/howto-guide-adding-generated-libraries.md

```diff
@@ -70,6 +70,14 @@ example, [here][retryable-status-codes].
 
 Manually edit `generator/generator_config.textproto` and add the new service.
 
+Find the list of `.proto` files that will need to be included:
+
+```shell
+bazel_output_base="$(bazel info output_base)"
```

Comment:
Nit: Perhaps this should be moved to the _"Set some useful variables"_ section now that it appears in multiple places?

Reply:
Done.